packages/accounts/src/kernel-zerodev/README.md

```diff
@@ -1,6 +1,6 @@
 # Zerodev Kernel Account API
 
-This module provides a Signer + Provider implementation of Zerodev's Kernel V2 on top of `@alchemy/aa-core`
+This module provides a signer + Provider implementation of Zerodev's Kernel V2 on top of `@alchemy/aa-core`
```

Comment:
is there a reason we're lowercasing Signer but keeping Provider uppercase? It's a primitive type definition in the aa-sdk, and so I think it should be capitalized here and elsewhere

Reply:
Capitalizing it now, this was one of the request in the task description that's why changed it to lowercase before